orderer/consensus/etcdraft/blockcreator.go

```diff
@@ -35,14 +35,21 @@ type blockCreator struct {
 // and it will be ignored during block data hash computation, but will still be carried along
 // in the subsequent flow of the system (unless someone is copying it manually and then we'll need to chase down
 // why it was stripped out...)
+// GAL: so this is where most of the orderer's roll is done? who calls this?
+// GAL: blk in new fmt
 func (bc *blockCreator) createNextBlock(envs []*cb.Envelope) *cb.Block {
 	data := &cb.BlockData{
 		Data: make([][]byte, len(envs)),
 	}
 
+	pis := make([][]byte, 100)
+
 	var err error
 	for i, env := range envs {
 		data.Data[i], err = proto.Marshal(env)
```

Comment:
I think we need to delete the PIS from the envelope before you marshal it, otherwise it will be impossible to redact later.